@ndlib/gatsby-theme-marble/src/assets/i18n/json/common.default.en.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Starting with some example content. The "common" namespace is where menu items, buttons, labels, etc will live. "Default" things live here.